lib/libutils/isoc/fp.c

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2020, Huawei Technologies Co., Ltd
+ */
+
+#include <stdio.h>
+
+struct _FILE {
+};
```

Comment:
What does this mean? I suppose the compiler adds a dummy field to make sense of this. How about being explicit and add a `char dummy` field?

Reply:
Fixed